client/src/main/java/io/pravega/client/tables/impl/TableSegment.java

```diff
@@ -37,7 +37,7 @@
 
     /**
      * Inserts or updates an existing Table Entry into this Table Segment.
-     * @param entry The Entry to insert or update. If {@link TableEntry#getKey()#getVersion()} is null, this will perform
+     * @param entry The Entry to insert or update. If {@link TableEntry#getKey()}#getVersion() is null, this will perform
```

Comment:
My understanding is that javadoc `@link` doesn't support method chaining. So to fix the issue, I have moved out `#getVersion()` from `@link`. Though not the right fix, I hope it is better than existing.

Reply:
Sure, that looks better. Should we use something like `{@link TableEntry#getKey()} {@link TableEntry#getVersion(...)}` instead? I think that'd clarify even better. What do you say?

Reply:
yeah.. that might be better. changed to `{@link TableEntry#getKey()}{@link TableKey#getVersion()}`